doc/classes/GDScriptLanguageProtocol.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<class name="GDScriptLanguageProtocol" inherits="JSONRPC" experimental="" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
+	<brief_description>
+		GDScript language server.
+	</brief_description>
+	<description>
+		Provides access to certain functionality that is implemented in the language server.
+		[b]Note:[/b] This class is not a language server client that can be used to access LSP functionality. It only provides access to a limited set of functionality
+		that is implemented using the same technical foundation as the language server.
```

Comment:
Each line break creates a new paragraph in the class reference, which means lines shouldn't be wrapped:

```suggestion
		[b]Note:[/b] This class is not a language server client that can be used to access LSP functionality. It only provides access to a limited set of functionality that is implemented using the same technical foundation as the language server.
```